Documentation/getting_started/4_adding_basic_code.md

```diff
@@ -29,21 +32,27 @@ Now run the game.
 ![Draw Ball 1](images/getting_started/4_ball_not_center.png)
 
 As you can see the ball doesn't seem quite centered yet. This is happening because MonoGame uses (0, 0) as the origin point for drawing by default. We can modify this by doing the following:
+
 ```csharp
 spriteBatch.Draw(
-    texture: ballTexture, 
-    position: ballPosition, 
-    color: Color.White,
-    origin: new Vector2(ballTexture.Width / 2,
-        ballTexture.Height / 2)
+    ballTexture,
+    ballPosition,
+    null,
+    Color.White,
+    0f,
+    new Vector2(ballTexture.Width / 2, ballTexture.Height / 2),
+    Vector2.One,
+    SpriteEffects.None,
+    0f
```

Comment:
I think it's good to actually be explicit with the parameter names here. It clarifies the meaning of the call.

Reply:
No, that method is deprecated.

Reply:
Yeah, I don't suggest reverting the change, but I suggest adding the parameter names.

Reply:
For now I'll just keep this.